components/message/Message.js

```diff
@@ -30,10 +31,10 @@ class Message extends PureComponent {
         <div className={theme['content']}>
           {children}
           {hasAction && (
-            <div className={theme['actions']}>
+            <ButtonGroup className={theme['actions']} marginTop={4}>
               {button && <span className={theme['button']}>{button}</span>}
```

Comment:
Do you think wrapping in a `span` is still necessary? What if we add the `class` to the `button` itself? 🤔

Reply:
The button itself is actually injected by a prop `button`. So we kinda need a wrapper to fix it's margins when changing direction to vertical.

Note: This is temporary because I'm refactoring the Box and ButtonGroup component to be responsible for the alignment and correct margins. Afterwards we can drop this wrapping span element.

Reply:
I was not talking about `ButtonGroup`, rather about the `span` that was already there wrapping `{button}`. I'm pretty sure you can achieve the same effect without having the `span` there by doing 
```jsx
{button && React.cloneElement(button, { className: theme['button'] })}
```

Reply:
I know you're talking about the span and not the ButtonGroup. And my point is that once the Box is refactored, I can drop this span entirely.

Reply:
I think eventual refactoring is no valid argument for writing worse code. Why would you not *always* want to write the most performant code possible when the solution is as easy as this? #notproudtobepartofteamleader with that way of thinking

Reply:
so deep, @InstaK 👏

Reply:
Just being passionate and wanting the best :) Thanks for changing it, now I can sleep at night ❤️ 

 :ok: by @InstaK on 14/12/2017 15:02:59